Source/WebKit/UIProcess/API/Cocoa/NSAttributedString.mm

```diff
@@ -144,6 +144,9 @@ + (WKWebViewConfiguration *)configuration
         [configuration _setAllowsJavaScriptMarkup:NO];
         [configuration _setAllowsMetaRefresh:NO];
         [configuration _setAttachmentElementEnabled:YES];
+#if HAVE(APP_SSO)
```

Comment:
nit: I believe we don't need this #if check. The SPI is available no matter what.